lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -93,6 +93,11 @@ public function execute(InputInterface $input, OutputInterface $output)
         $path = $this->generateMigration($configuration, $input, $version);
 
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>"', $path));
+
+        $commandName = sprintf('%s:execute', $this->getApplication()->extractNamespace($this->getName()));
+        if (count($command = $this->getApplication()->find($commandName))) {
+          $output->writeln(sprintf('To run just this migration for testing purposes, you can use "<info>%s --up %s</info>"', $command->getName(), $version));
```

Comment:
Can you change the indention here to 4 spaces instead of 2.

Reply:
Also see https://github.com/doctrine/migrations/pull/174/files#r60681298.